src/main/webapp/app/course/course-access-storage.service.ts

```diff
@@ -5,25 +5,28 @@ import { LocalStorageService } from 'ngx-webstorage';
     providedIn: 'root',
 })
 export class CourseAccessStorageService {
-    private static readonly STORAGE_KEY = 'artemis.courseAccess';
+    public static readonly STORAGE_KEY = 'artemis.courseAccess';
+    public static readonly STORAGE_KEY_DROPDOWN = 'artemis.courseAccessDropdown';
+    public static readonly MAX_RECENTLY_ACCESSED_COURSES_OVERVIEW = 3;
+    public static readonly MAX_RECENTLY_ACCESSED_COURSES_DROPDOWN = 6;
```

Comment:
Nitpick: I think the code would be easier to read if we add a "displayed" in the variable name

```suggestion
    public static readonly MAX_DISPLAYED_RECENTLY_ACCESSED_COURSES_OVERVIEW = 3;
    public static readonly MAX_DISPLAYED_RECENTLY_ACCESSED_COURSES_DROPDOWN = 6;
```

Reply:
thats not fully correct as only 5 courses will be displayed. Thats why the different naming 👍

Reply:
I think that this is rather confusing, I do not really get what the variable expresses then 😅

Reply:
ok i added a comment 👍